src/java.base/share/classes/java/lang/foreign/MemoryLayout.java

```diff
@@ -625,12 +625,12 @@ public sealed interface MemoryLayout
      *     (this layout), or an {@link IllegalArgumentException} is thrown. Note
      *     that the alignment constraint of the root layout can be more strict
      *     (but not less) than the alignment constraint of the selected value layout.</li>
-     *     <li>The offset of the access operation (computed as above) must fall inside
-     *     the spatial bounds of the accessed memory segment, or an
-     *     {@link IndexOutOfBoundsException} is thrown. This is the case when
-     *     {@code O + A <= S}, where {@code O} is the accessed offset (computed as above),
-     *     {@code A} is the size of the selected layout and {@code S} is the size of the
-     *     accessed memory segment.</li>
+     *     <li>The access operation must fall inside the spatial bounds of the accessed
+     *     memory segment, or an {@link IndexOutOfBoundsException} is thrown. This is the case
+     *     when {@code B + A <= S}, where {@code O} is the base offset (defined above),
```

Comment:
Do you mean `{@code O + A <= S}`?
(Still working my way through the changes...)

Reply:
Yes, apologies